packages/api/lambdas/bulk-operation.js

```diff
@@ -78,8 +78,11 @@ async function applyWorkflowToGranules({
  */
 async function bulkGranuleDelete(
   payload,
-  unpublishGranuleFunc = unpublishGranule
+  removeGranuleFromCmrFunction
```

Comment:
Reminder to udpate the docstring

Reply:
Also is it weird that this could be undefined? I see that we're assigning a default in `unpublishGranule` and that's the only place it's used so maybe not 🤔

Reply:
Docstring updated: 60ebe10f96